add missing include statement to fix problem with std::clog not being found
author: Kenneth Hoste (HPC-UGent)
--- bcl2fastq/src/cxx/include/common/Logger.hh.orig	2017-12-05 17:48:50.023844797 +0100
+++ bcl2fastq/src/cxx/include/common/Logger.hh	2017-12-05 17:49:01.894010324 +0100
@@ -16,6 +16,7 @@
 
 
 #include <ios>
+#include <iostream>
 #include <string>
 
 #include <boost/atomic.hpp>
